pyanaconda/modules/runtime/dracut_only_commands/dracut_only_commands.py

```diff
@@ -0,0 +1,83 @@
+#
+# The user interface module
+#
+# Copyright (C) 2023 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+from collections import namedtuple
+from pyanaconda.modules.common.base import KickstartBaseModule
+
+__all__ = ["DracutOnlyCommandsModule"]
+
+
+DriverDiskLocalData = namedtuple("DriverDiskLocalData", ["partition", "source", "biospart"])
+SshPwLocalData = namedtuple("SshPwLocalData", ["username", "is_crypted", "password", "lock"])
+
+
+class DracutOnlyCommandsModule(KickstartBaseModule):
+    """The dracut-only commands module.
+
+    This module only stores data for these commands during the anaconda session. The data is not
+    externally accessible. Thus, this module is also interface-less.
+    """
+    def __init__(self):
+        super().__init__()
+        self._driverdisks = []
+        self._ssh_pws = []
+        self._mediacheck = False
+        self._updates_url = ""
+
+    def process_kickstart(self, data):
+        """Process the kickstart data."""
+        for ks_dd_data in data.driverdisk.driverdiskList:
```

Comment:
I would be fine with keeping and reusing a copy of the `DriverDiskData` list if you want to simplify the code. The data objects shoudn't have a reference to the kickstart handler (unlike the command objects). Same for `SshPwData`.

Reply:
I'm conflicted. Keeping the data as such did not appear to be a good idea in my previous testing. On the other hand, missing potential new options here would be a problem, too.